resqpy/grid/xyz.py

```diff
@@ -179,13 +178,11 @@ def global_to_local_crs(grid,
     # todo: check resqml definition for order of rotation and translation and apply rotation if not zero
     if global_xy_units is not None:
         crs_xy_units_text = rqet.find_tag(crs_root, 'ProjectedUom').text
-        if crs_xy_units_text in ['ft', 'm']:  # todo: else raise exception
```

Comment:
I've already made changes to global_to_local_crs() but I can resolve the conflicts